src/olympia/devhub/tests/test_views_submit.py

```diff
@@ -675,6 +675,18 @@ def test_submit_source_targz(self):
             oct(os.stat(self.get_version().source.path)[stat.ST_MODE]))
         assert mode == '0100644'
 
+    def test_submit_source_tgz(self):
+        response = self.post(
+            has_source=True, source=self.generate_source_compressed_tar(
```

Comment:
I think `generate_source_compressed_tar` only passes in `w:bz2` or `w:gz` and this should require a regular, uncompressed `w` mode.

Reply:
No ? `.tgz` *should* be compressed with `w:gz`, it's a `.tar.gz` with a shorter suffix, that's all.

Reply:
eh, apologies, I accidentally read `.tar` and not `.tgz` for some reason, all good!